OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/keyunlock/Model/WizardModel.java

```diff
@@ -0,0 +1,96 @@
+package org.sufficientlysecure.keychain.ui.keyunlock.Model;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import org.sufficientlysecure.keychain.pgp.CanonicalizedSecretKey;
+import org.sufficientlysecure.keychain.util.Passphrase;
+
+import java.util.ArrayList;
+
+/**
+ * Wizard Data Model
+ */
+public class WizardModel implements Parcelable {
+    private String mName;
+    private String mEmail;
+    private ArrayList<String> mAdditionalEmails;
+    private Passphrase mPassword;
+    private CanonicalizedSecretKey.SecretKeyType mSecretKeyType;
+
+    public String getName() {
+        return mName;
+    }
+
+    public void setName(String mName) {
+        this.mName = mName;
```

Comment:
fixed, my bad, didn't had the prefix properly configured on android studio and missed those.